src/node_postmortem_metadata.cc

```diff
@@ -44,14 +44,6 @@
 #include <utility>
```

Comment:
We can remove those includes as well since they are only here to avoid problems with private class inheritance